README.md

```diff
@@ -1,2 +1,45 @@
 # OC-Test
 OpenShift Testing
+
+Pipeline setup steps
+
+1) Set up Jenkins project
+
+##create new project named test-app-pipeline
+oc new-project test-app-pipeline 
+##set up an app from jenkins-ephemeral template
+oc new-app jenkins-ephemeral 
+##to view available templates, run 'oc new-app --search jenkins'
+##apply the build config with name esri-example-pipeline to jenkins app (NOTE: replace {REPO_GITHUB_ADDR} with github repository)
+oc create -f Pipeline/jenkinsBuildConfig.yaml 
+##get route for Jenkins web console
+oc get route 
+##go to the route url to access Jenkins Web Console; on first log in, give Jenkins requested permissions
+
+2) Set up project to deploy C++ application using s2i
```

Comment:
Step 2 should be done before Setting up Jenkins

Reply:
Not sure if it matters - I followed the steps in Chapter 8, section 2 of the do288 coursebook. Can change if it seems weird.